### PR TITLE
Integrate postgres as subchart

### PR DESCRIPTION
**Pull request by sliverc**
Postgres subchart is optional and can be disabled by postgres.install. This PR also changes naming of helpers so they do not clash with subcharts. For completeness a admin user will be created with given or random password.

### CODE REVIEW

**Comment by sliverc** [comment]
@tongpu Added notes how to retrieve password for user to login. Ready for review.

File: templates/NOTES.txt [hunk 1]
```diff
@@ -1,28 +1,19 @@
...
+  kubectl delete job post-upgrade-{{ template "timed.fullname" . }}-dbmigrate
 {{- end }}
 {{- if .Release.IsInstall }}
-  kubectl delete job post-install-{{ template "fullname" . }}-dbmigrate
+  kubectl delete job post-install-{{ template "timed.fullname" . }}-dbmigrate
 {{- end }}
 
 2. Open following URL to start using Timed:
 {{- if .Values.ingress.enabled }}
 {{- range .Values.ingress.hosts }}
```
Comment (hairmare) [inline templates/NOTES.txt:12]: This `range` seems to have lost its `end`:

```
Error: UPGRADE FAILED: parse error in "timed-chart/templates/NOTES.txt": template: timed-chart/templates/NOTES.txt:12: unexpected EOF
```

Apart from that I seem to have broken my helm setup so I can't currently do a complete test of the changes. I'll do so as soon as I've figured out how to repair a broken helm on ACS.

File: templates/NOTES.txt [hunk 2]
```diff
@@ -1,28 +1,19 @@
 1. In Helm Version <2.7 following job needs to be cleaned manually.
```
Comment (hairmare) [inline templates/NOTES.txt:1]: Also, this still seems to be needed with helm 2.7.

**Review by hairmare — CHANGES_REQUESTED** [review]
See my individual comments for the bugs I encountered. Other than that I got everything to deploy properly on ACS :)

File: templates/NOTES.txt (hunk 1 above)
Comment (sliverc) [reply templates/NOTES.txt:12]: Fixed.

File: templates/NOTES.txt (hunk 2 above)
Comment (sliverc) [reply templates/NOTES.txt:1]: We need to verify this as Helm 2.7 includes support of `"helm.sh/hook-delete-policy"` and should clean up automatically.

**Review by hairmare — APPROVED** [review]

**Pull request merged**